test/credentials_test.dart

```diff
@@ -299,4 +299,185 @@ Future<void> main([List<String>? args]) async {
       await authProvider.callResetPasswordFunction(username, newPassword);
     }, throws<RealmException>("failed to reset password for user $username"));
   }, appName: AppNames.autoConfirm);
+
+  /// JWT Header:
+  /// {
+  ///   "kid": "1",
+  ///   "alg": "RS256",
+  ///   "typ": "JWT"
+  /// }
```

Comment:
The header is rarely interesting, so I don't think we need to include it in the comments.

Reply:
I think these both are important:
"kid": "1",
"alg": "RS256"
But I can remove them.

Reply:
Removed.